src/test/java/edu/ucsb/cs156/happiercows/controllers/UserCommonsControllerTests.java

```diff
@@ -44,40 +44,42 @@
 @AutoConfigureDataJpa
```

Comment:
If this is only whitespace/formatting changes, PLEASE DON'T DO THIS.

It makes your PR super hard to code review, because we have to look at every blessed line to make sure you didn't delete or add something.

Fixing formatting should be done in a separate PR that ONLY focuses on fixing formatting.